application/src/main/kotlin/net/corda/v5/application/flows/RPCStartableFlow.kt

```diff
@@ -0,0 +1,30 @@
+package net.corda.v5.application.flows
+
+import net.corda.v5.base.annotations.Suspendable
+
+/**
+ * A flow that may be started via RPC.
+ *
+ * Flows initiated due to an RPC request receive a JSON encoded string in their call method, containing the body of the
+ * RPC request that initiated this flow. The JSON marshalling service may be used to convert this input string into a
+ * concrete object. The string return type is treated by the platform as a JSON encoded string to return to the RPC
+ * service, and will appear in the RPC flow status when the flow completes. Again, this can be generated by serializing
+ * via the JSON marshalling service.
+ *
+ * Flows implementing this interface must have a no-arg constructor. The flow invocation will fail if this constructor
+ * does not exist.
+ */
+interface RPCStartableFlow {
```

Comment:
I know @mnesbit favours a slightly different approach here where we give two functions - one to set the parameters on the flow, and the current `call` function but with no parameters. I think the argument is that it gives the user the option to throw away some bits of the input request, as otherwise it's always in the checkpoint. I think that API is less clear though. Thoughts?

Reply:
The wrapper handles stuff for now I think